dbhydra/dbhydra_core.py

```diff
@@ -1287,17 +1281,39 @@ def get_all_columns(self):
 
     def get_all_types(self):
 
+        only_types, lengths = self.get_data_types_and_cahracter_lenghts()
+        for i in range(len(only_types)):
+            if lengths[i] is not None:
+                only_types[i] = only_types[i] + f"({lengths[i]})"
+        return (only_types)
+
+    def get_data_types_and_cahracter_lenghts(self):
         information_schema_table = Table(self.db1, 'INFORMATION_SCHEMA.COLUMNS', ['DATA_TYPE'], ['nvarchar(50)'])
         query = f"SELECT DATA_TYPE,character_maximum_length FROM INFORMATION_SCHEMA.COLUMNS WHERE TABLE_SCHEMA = '{self.db1.DB_DATABASE}' AND TABLE_NAME  = '" + self.name + "'"
         types = information_schema_table.select(query)
         only_types = [x[0] for x in types]
         lengths = [x[1] for x in types]
```

Comment:
I urged to change this, why is it still here? "only_types" is really stupid name when you subset it from "types"... - not mentioning this is quite deep change of DBhydra functionality, I would expect some discussion about it